actix/src/sec-5-arbiter.md

```diff
@@ -15,89 +15,148 @@ for your actors to run inside. When you call `start()` on your actor it is then
 running inside of the System Arbiter's thread. In many cases, this is all you
 will need for a program using Actix.
 
+While it only uses one thread, it uses the very efficient event loop pattern
+which works well for asynchronous events. To handle synchronous tasks that
+require heavy CPU power, it's better to avoid blocking the event loop and
+instead offload the computation to other threads. For this usecase, read the
+next section and consider using [`SyncArbiter`](./sec-6-sync-arbiter.md).
+
+## The event loop
+
+> If you're familiar with event loops in other paradigms, such as libuv (upon
```

Comment:
So I thought about this for a few hours, and I don't like this bit about libuv, node, tokio or js. I think that it says a lot of words, but communicates little. If the person has used these languages they will already find it familiar, we don't need to spell that out to them. For people who aren't familiar, they may feel like they are not good enough or experienced enough to use these concepts since they are not familiar.

So I think these two paragraphs should be removed, and this document should remain factual and example oriented.

Reply:
I struggle with this because I agree with you, but it would have saved a BOATLOAD of conceptual time for me as a JS dev if someone had just come out and said these two paragraphs.

I think you (or maybe someone else?) are writing a doc on converting from Akka. Maybe a set of "Coming from..." docs would be helpful.

Reply:
No idea why I thought this was you: https://github.com/actix/book/pull/18

Regardless, I think basically a folder for parallels to other libraries could be very helpful. It's way harder to explain what a `Box` is from scratch than it is to say "it's like a `uniq_ptr`", y'know?

Reply:
Yep, not me :) my background is all in C and concurrent databases. 

I still think it's not needed, because Box isn't really like a unique_ptr - the analogy is nice, but also incorrect as box and unique_ptr are quite different. For the people who read it and understand, they'll know how to use it. I stand by the request to remove this part.